docs/ref/django-admin.txt

```diff
@@ -1543,6 +1543,7 @@ Use the ``--locale`` or ``-l`` option to specify the locale to process.
 If not provided all locales are processed.
 
 .. django-admin-option:: --noinput
+.. django-admin-option:: --no-input
```

Comment:
Need a `.. versionchanged:: 1.9` annotation at the end of this section noting that the alias was added. A mention in the 1.9 release notes is also appropriate. Thanks!